scripts/sysman/EXT_MemoryBwCounterValidBits.rst

```diff
@@ -0,0 +1,28 @@
+<%
+import re
+from templates import helper as th
+%><%
+    OneApi=tags['$OneApi']
+    x=tags['$x']
+    X=x.upper()
+    s=tags['$s']
+    S=s.upper()
+%>
+:orphan:
+
+.. _ZES_extension_mem_bw_counter_valid_bits:
+
+==============================================
+ Memory Bandwidth Counter Valid Bits Extension
+==============================================
+
+API
+----
+
+* Enumerations
+
+    * ${s}_mem_bw_counter_bits_exp_version_t
```

Comment:
It may be better to use "bandwidth" instead of "bw", so that we are following similar naming conventions as `s_mem_bandwidth_t`

Reply:
Corrected.